app/views/spree/admin/products/index/_products_variant.html.haml

```diff
@@ -29,6 +29,6 @@
   %td.actions
     %a{ 'ng-click' => 'editWarn(product,variant)', :class => "edit-variant icon-edit no-text", 'ng-show' => "variantSaved(variant)", 'ofn-with-tip' => t(:edit)  }
   %td.actions
-    %span.icon-warning-sign{ 'ng-if' => 'variant.variant_overrides', 'ofn-with-tip' => "{{ 'spree.admin.products.index.products_variant.variant_has_n_overrides' | t:{n: variant.variant_overrides.length} }}" }
+    %span.icon-warning-sign{ 'ng-if' => 'variant.variant_overrides_count > 0', 'ofn-with-tip' => "{{ 'spree.admin.products.index.products_variant.variant_has_n_overrides' | t:{n: variant.variant_overrides_count} }}" }
```

Comment:
it'd be even better with https://apidock.com/rails/ActiveRecord/Base/exists%3f/class

Reply:
I don't understand how that would be used..?

At this point in the view, the value is an integer that was serilialized previously and is now in a javascript object, we're not querying the actual variant record.

Reply:
nevermind, I missed this was a JS object and not an AR model.